webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -422,6 +424,7 @@ class InterpolatedValueCompilerImpl extends InterpolatedValueCompiler {
       case NonRudderVar(s)    => s"$${${s}}"
       case NodeAccessor(path) => s"$${rudder.node.${path.mkString(".")}}"
       case Param(name)         => s"$${rudder.param.${name}}"
+      case RudderEngine(e, m, p) => s"$${rudder.engine[${e}${m.mkString("][")}(${p.head._1})}"
```

Comment:
Cf `param` implementation above for `${p.head._1}`

Reply:
I think it should be an error, because it won't be a valid format (cfengine or dsc). 
We will need to decide what to do with that. Perhaps use something easy to debug like: 
```
s"[missing value for: $${rudder.engine[${e}${m.mkString("][")}](${p.toCorrectSerialisationMethodWeWantHere})}]"
```
(also you were missing a closing `]`